src/uucore/src/lib/features/checksum.rs

```diff
@@ -0,0 +1,40 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use std::cmp::Ordering;
+
+use crate::show_warning_caps;
+
+pub fn cksum_output(bad_format: i32, failed_cksum: i32, failed_open_file: i32) {
+    match bad_format.cmp(&1) {
+        Ordering::Equal => {
+            show_warning_caps!("{} line is improperly formatted", bad_format);
+        }
+        Ordering::Greater => {
+            show_warning_caps!("{} lines are improperly formatted", bad_format);
+        }
+        Ordering::Less => {}
+    };
```

Comment:
Is a `match` really the right thing to do here? In my eyes, `if` is shorter and easier to read in this case.
```suggestion
    if bad_format == 1 {
        show_warning_caps!("{} line is improperly formatted", bad_format);
    } else if bad_format > 1 {
        show_warning_caps!("{} lines are improperly formatted", bad_format);
    }
```